core/scripts/server.js

```diff
@@ -125,10 +125,38 @@ app.post('/invalidate', invalidateCache)
 app.get('/invalidate', invalidateCache)
 
 app.get('*', (req, res, next) => {
+  const contains = (target, pattern) => {
+    var value = 0
+    pattern.forEach(word => {
+      value = value + target.includes(word)
+    })
+    return (value >= 1)
+  }
+
+  if (contains(req.url, config.server.ssrDisabledFor.extensions)) {
+    return
+  }
+
   const s = Date.now()
   const errorHandler = err => {
     if (err && err.code === 404) {
-      res.redirect('/page-not-found')
+      if (contains(req.url, config.server.customErrors.http404.notAllowedExtensions)) {
```

Comment:
here again would be
```suggestion
      if (NOT_ALLOWED_SSR_EXTENSIONS_REGEX.test(req.url)) {
```

or another const if this other setting from config is really needed here

Reply:
I like this suggestion. will do with another const for now. then they could be merged  later

Reply:
Done.